addons/website_sale_digital/product.py

```diff
@@ -5,4 +5,56 @@
 class product_template(models.Model):
     _inherit = ['product.template']
 
-    digital_content = fields.Boolean('Digital Content', help="If checked, it will allow clients to download the product attachments when they have bought it.")
+    type = fields.Selection([('digital', 'Digital Content'),('product', 'Stockable Product'), ('consu', 'Consumable'), ('service', 'Service')], help="Consumable: Will not imply stock management for this product. \nStockable product: Will imply stock management for this product.\n Digital Product :If select digital , it will allow clients to download the product attachments when they have bought it.")
```

Comment:
Should have been done with an inherited method. Not hardcoding value, screwing inheritance.